tests/StorageBackendTest.php

```diff
@@ -2,76 +2,36 @@
 
 namespace Keboola\ManageApiTest;
 
-use Keboola\ManageApi\Backend;
 use Keboola\ManageApi\ClientException;
+use Keboola\StorageApi\Workspaces;
 
 class StorageBackendTest extends ClientTestCase
```

Comment:
Nemělo by to mít setUp s nějakým cleanupem ?

Reply:
Když ten testCreateStorageBackend selže někde v půlce, tak tam ten backend zůstane IMHO

Reply:
Chcel som to ale z toho $this->client->listStorageBackend(); nejde nejak dobre urcit to podla cohoby som to mohol bezpecne mazat :/

Reply:
Fair enough